File: guilherme/Problema0298/Problema0298.py
vetor = list(range(10))
x = 1
maior = 0
print("Digite uma sequência de 10 números")
for i in range(10):
    vetor[i] = int(input())
    if(i == 0 or (i > 0 and vetor[i] < vetor[i-1])):
        x = 1
    elif(i > 0 and vetor[i] > vetor[i-1]):
        x += 1
    if(x > maior):
        maior = x
print("Maior Sequência:",maior)

File: guilherme/Problema0298/Problema0298.java
import java.util.Scanner;

public class Problema0298{
    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        int[] vetor = new int[10];
        int x = 1, maior = 0;
        System.out.println("Digite uma sequência de 10 números");
        for(int i = 0; i < 10; i++){
            vetor[i] = Integer.valueOf(input.nextLine());
            if(i == 0 || (i > 0 && vetor[i] < vetor[i-1]))
                x = 1;
            else if(i > 0 && vetor[i] > vetor[i-1])
                x++;
            if(x > maior)
            maior = x;
        }
        System.out.println("Maior Sequência: "+maior);
        
    }
}